marketpulse/main/templates/fxosprice_all.html

```diff
@@ -28,11 +28,14 @@ <h3 class="title">Your contributions</h3>
               {% endif %}
             </td>
             <td>{{ contribution.location.shop_name }}</td>
-            <td class="delete-contribution">
-              <button class="btn btn-danger btn-xs ask" href="#">Delete</button>
-              <span class="delete-confirm">
-                <a class="btn btn-danger btn-xs" href="{{ url('main:delete_fxosprice', contribution.pk) }}">Yes</a>
-                <a class="btn btn-default btn-xs cancel" href="#">Cancel</a>
+            <td class="contribution-actions">
+              <a class="btn btn-warning btn-xs" href="{{ url('main:edit_fxosprice', contribution.pk) }}">Edit</a>
+              <span class="delete-contribution">
+                <button class="btn btn-danger btn-xs ask" href="#">Delete</button>
+                <span class="delete-confirm">
+                  <a class="btn btn-danger btn-xs" href="{{ url('main:delete_fxosprice', contribution.pk) }}">Yes</a>
+                  <a class="btn btn-default btn-xs cancel" href="#">Cancel</a>
+                </span>
```

Comment:
With the last merge this is replaced by a `details` button and all contribution actions are moved inside the single view. Please move all of these to their new placement.